chapters/functions.tex

```diff
@@ -1644,6 +1645,32 @@ \section{Declaring Inverses of Functions}\doublelabel{declaring-inverses-of-func
 \end{lstlisting}
 \end{example}
 
+The sub-set case is useful if $f_1$ computes the inverse of $f_2$ within a region, or up to a certain tolerance,
+the $f_1$ may specify $f_2$ as inverse with fewer arguments, skipping the arguments for tolerance and/or the region.
```

Comment:
Long sentence that is hard to parse.  What about this?
> The sub-set case is useful if $f_1$ computes the inverse of $f_2$ within a region, or up to a certain tolerance.  Then, $f_1$ may specify $f_2$ as inverse with fewer arguments, skipping the arguments for tolerance and/or the region.

Reply:
Should be resolved now.